pyrometheus/codegen/fortran.py

```diff
@@ -259,10 +259,10 @@ def map_if(self, expr, enclosing_prec):
     ${real_type}, parameter :: gas_constant = ${float_to_fortran(ct.gas_constant)}
     ${real_type}, parameter :: molecular_weights(${sol.n_species}) = &
         (/ ${str_np(sol.molecular_weights)} /)
-    ${real_type}, parameter :: inv_molecular_weights(${sol.n_species}) = &
+    ${real_type}, parameter :: inv_weights(${sol.n_species}) = &
```

Comment:
This change makes the Fortran template inconsistent (wrt to the other templates) in the naming of this variable. Please investigate further why the compiler did not like `inv_molecular_weights`. As discussed in person, it is likely that the problem is no the array name, but how it was used inside a `sum` (line 413).